nari/ext/act/utils.py

```diff
@@ -12,33 +12,42 @@
 
 
 # pylint: disable=invalid-name
-class ActLogChecksumType(Enum):
+class ActLogChecksumType(IntEnum):
     """List of hashsum algorithms used by different ACT versions"""
-    MD5 = "md5"
-    SHA256 = "sha256"
+    MD5 = 0
```

Comment:
Is it worth using `auto` from enum? Docs [here](https://docs.python.org/3/library/enum.html#using-automatic-values)

Reply:
These needs to be locked to be known for chceking between rust and python.

Reply:
Auto is just like a C enum. It starts at zero and increments.

Reply:
Will rather have them hard coded than have to guess